desktop/release-notes.md

```diff
@@ -24,34 +24,109 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.17.0
+
+{% include release-date.html date="2022-02-27" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Image analysis results - you can now filter on vulnerability information.
```

Comment:
I think these first four bullet points all relate to Docker Scout. Think we should reduce it to just `Docker Desktop now ships with Docker Scout. View and pull images from Artifactory repositories, get base image updates and recommended tags and digests, and filter your images on vulnerability information. To learn more, see [Docker Scout](/scout/index.md)`

Reply:
@dvdksn am I right?

Reply:
Collapsing the bullets is fine, but I would just add Hub here for clarity:

> Docker Desktop now ships with Docker Scout. Pull and view analysis for images from **Docker Hub and** Artifactory repositories, get base image updates and recommended tags and digests, and filter your images on vulnerability information. To learn more, see [Docker Scout](/scout/index.md)

Reply:
@aevesdocker I'm not sure we have docs for `Docker Scout` yet 🤔 Is there another PR with `/scout/index.md` ?

Reply:
yes :) https://github.com/docker/docs/pull/16679/files

Reply:
The build will fix itself when the Scout PR is merged.